web-api/terraform/applyables/blue/blue.tf

```diff
@@ -20,7 +20,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 5.89.0"
+      version = "5.92.0"
```

Comment:
Not a problem, just worth noting for the next set of dep updates that touches this (since it won't match the regex in `dependency-updates.md`